source/documentation/how-we-work.html.md.erb

```diff
@@ -109,7 +109,30 @@ The [MOJ Forms Tech Docs](https://ministryofjustice.github.io/moj-forms-tech-doc
 The first port of call for users of Formbuilder is the [#ask-formbuilder](https://mojdt.slack.com/archives/CE26QEL1Z) Slack channel. This channel will be monitored by the Product Managers who will contact the support dev as required.
 
 ### Alerts
-We have two alerts channels, [#form-builder-alerts](https://mojdt.slack.com/archives/CH4FJB8E4) and [#form-builder-alerts-low-severity](https://mojdt.slack.com/archives/C013L7PA4N4).
+We have three alerts channels, [#form-builder-alerts](https://mojdt.slack.com/archives/CH4FJB8E4), [#form-builder-alerts-low-severity](https://mojdt.slack.com/archives/C013L7PA4N4) and [#form-builder-alerts-400s](https://mojdt.slack.com/archives/C02TQKB1810).
+
+The #form-builder-alerts-400s is used for all the 400s we get from the `formbuilder-services-live-production` namespace. This was added because we our services are constantly probed by bots but we do not necessarily need to act on every 400 alert that happens. We do need to keep an eye on the number of 400s over time and respond accordingly but the high severity alert channel is really for something that has gone absolutely wrong. Too many alerts will desensitise the developers. The high severity channel contains people who are stakeholders for the platform but not necessarily developers.
```

Comment:
```suggestion
The #form-builder-alerts-400s is used for all the 400s we get from the `formbuilder-services-live-production` namespace. This was added because our services are constantly probed by bots but we do not necessarily need to act on every 400 alert that happens. We do need to keep an eye on the number of 400s over time and respond accordingly but the high severity alert channel is really for something that has gone absolutely wrong. Too many alerts will desensitise the developers. The high severity channel contains people who are stakeholders for the platform but not necessarily developers.
```

Reply:
just this one small thing but otherwise looks good to me 🚀

Reply:
good eyes! have update, thanks